### ISSUE TITLE
undefined local variable or method `conn'

### ISSUE THREAD

**Issue by bhb** [jstorimer/spin#28]
When using --push-results I get an error

```
$ spin serve -Itest --test-unit --push-results
Preloaded Rails env in 2.5115768909454346s...
Pushing test results back to push processes
/Users/ben/.rvm/gems/ruby-1.9.2-p290@reader2/gems/spin-0.4.1/bin/spin:153:in `block in fork_and_run': undefined local variable or method `conn' for main:Object (NameError)
        from /Users/ben/.rvm/gems/ruby-1.9.2-p290@reader2/gems/spin-0.4.1/bin/spin:148:in `fork'
        from /Users/ben/.rvm/gems/ruby-1.9.2-p290@reader2/gems/spin-0.4.1/bin/spin:148:in `fork_and_run'
        from /Users/ben/.rvm/gems/ruby-1.9.2-p290@reader2/gems/spin-0.4.1/bin/spin:122:in `block in serve'
        from /Users/ben/.rvm/gems/ruby-1.9.2-p290@reader2/gems/spin-0.4.1/bin/spin:98:in `loop'
        from /Users/ben/.rvm/gems/ruby-1.9.2-p290@reader2/gems/spin-0.4.1/bin/spin:98:in `serve'
        from /Users/ben/.rvm/gems/ruby-1.9.2-p290@reader2/gems/spin-0.4.1/bin/spin:245:in `<top (required)>'
        from /Users/ben/.rvm/gems/ruby-1.9.2-p290@reader2/bin/spin:19:in `load'
        from /Users/ben/.rvm/gems/ruby-1.9.2-p290@reader2/bin/spin:19:in `<main>
```

**Comment by jstorimer** [comment]
Fixed in v0.4.2.

**Comment by bhb** [comment]
Cheers
